roles/pulp_devel/templates/alias.bashrc.j2

```diff
@@ -44,7 +44,7 @@ _pdbreset_help="$_dbreset_help - `setterm -foreground red -bold on`THIS DESTROYS
 
 pclean() {
     pdbreset
-    sudo rm -rf {{ pulp_user_home }}/artifact
+    sudo rm -rf {{ pulp_media_root }}/*
```

Comment:
I believe there are other folders in the media root which ought to be cleaned up too (eg uploads).